pom.xml

```diff
@@ -219,7 +219,7 @@ THE SOFTWARE.
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-deploy-plugin</artifactId>
-          <version>2.7</version>
```

Comment:
Should probably be taking this version from [the parent POM](https://github.com/jenkinsci/pom/blob/6e2bdcc2b079c41a096f379abbc1dedfce2ff380/pom.xml#L967).